.github/workflows/functional-test.yaml

```diff
@@ -575,6 +577,15 @@ jobs:
           BICEP_RECIPE_REGISTRY: ${{ env.BICEP_RECIPE_REGISTRY }}
           BICEP_RECIPE_TAG_VERSION: ${{ env.BICEP_RECIPE_TAG_VERSION }}
           GH_TOKEN: ${{ steps.get_installation_token.outputs.token }}
+          GOTESTSUM_OPTS: '--junitfile ./dist/functional_test/results.xml'
+      - name: Process Functional Test Results
+        uses: ./.github/actions/process-test-results
+        # In case of failure, upload functional_test_results to artifacts so that they are not erased by subsequent runs.
```

Comment:
If I make the condition always the results captured along with test duration could identify some tests that take long. Would this be useful or is it a better idea to make that change as and when we are troubleshooting?

Reply:
There's basically no cost to adding something to artifacts. 

If you do use `always()` then think about what happens when the files don't exist.